src/data/eth-upgrade-articles.ts

```diff
@@ -1,4 +1,11 @@
-export const dannyArticles = [
+interface Article {
+  title: string
+  description: string
+  link: string
+}
+
+/** Articles by Danny Ryan (Ethereum Foundation) */
+export const dannyArticles: Article[] = [
```

Comment:
We are using the `Array<Article>` form. 
```suggestion
export const dannyArticles: Array<Article> = [
```

Reply:
👍